api/types.proto

```diff
@@ -277,6 +277,41 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
```

Comment:
Good flag name, but seems like not a great type/field name. Maybe `FailurePolicy`? Just a nit.